paddle/majel/malloc.cc

```diff
@@ -0,0 +1,131 @@
+#include "malloc.h"
+#include <glog/logging.h>
+
+#ifndef PADDLE_ONLY_CPU
+#include <cuda_runtime.h>
+#endif
+
+#define CHECK_CUDA(cudaFunc)                                         \
```

Comment:
I remember that @reyoung  and @helinwang both warning some time ago that PaddlePaddle as a library mustn't fatal with error, but need to return the error and make sure that it can be handled by the caller.  I think it's worthy to confirm with them.